docker-compose.yml

```diff
@@ -66,6 +66,7 @@ services:
     environment:
       - DSHOST=store
       - DSPORT=50050
+      - PROXY_IP=${PROXY_IP:-}
```

Comment:
Although this works, maybe it is better to set the default to the default value for the parameter (127.0.0.1), see here:
https://docs.gunicorn.org/en/stable/settings.html#forwarded-allow-ips

Reply:
Also, did you check if using env variables in the `CMD` of the dockerfile works at all?
According to this, it doesn't:
https://stackoverflow.com/a/40454758

Reply:
I tried to run the server, but it worked by accident. Do we need to use the list notation for the lunch CMD? If not, I have pushed a fix, that I have verified to work.

Reply:
The linting seems unhappy... How about the approach here with `"echo $VAR"`:
https://docs.docker.com/reference/dockerfile/#variable-substitution

It is probably the minimum change. Assuming it works...

Reply:
Also, I would just put the default in the docker-compose file, not also in the dockerfile `CMD`.

Reply:
It now work, used the method you linked.

Reply:
It works, but now the gunicorn doesn't get the proper shutdown command on restart/updates. It eventually just gets killed instead. This means it is slower to restart, but more importantly, requests might be stopped halfway.

I need to look at this a bit more... it seems like such a simple change.

Reply:
You can just set the `FORWARDED_ALLOW_IPS` in the env. I think this is the best way to do it. If left unset, it will just be set to default.

Reply:
Oh man... that is so much better. Have we been going in circles...

Only remaining question, do we want to drop the `DEBUG` loglevel now?

Reply:
Yes, ofcourse. I just added it to see if the changes took effect. Will remove it.